src/OrchardCore.Modules/OrchardCore.Twitter/Startup.cs

```diff
@@ -47,7 +48,17 @@ public override void ConfigureServices(IServiceCollection services)
 
             services.AddHttpClient<TwitterClient>()
                 .AddHttpMessageHandler<TwitterClientMessageHandler>()
-                .AddTransientHttpErrorPolicy(policy => policy.WaitAndRetryAsync(3, attempt => TimeSpan.FromSeconds(0.5 * attempt)));
+                .AddResilienceHandler("oc-handler", builder => builder
+                    .AddRetry(new HttpRetryStrategyOptions
+                    {
+                        Name = "oc-retry",
+                        MaxRetryAttempts = 3
+                    })
+                    .AddTimeout(new HttpTimeoutStrategyOptions
+                    {
+                        Name = "oc-timeout",
+                        Timeout = TimeSpan.FromSeconds(4),
```

Comment:
Same here. 4 seconds is different than the existing value. Not sure if this is okay or not. Probably is fine but not sure.